### ISSUE TITLE
Add default index pattern

### ISSUE THREAD

**Issue by DotNetRockStar** [elastic/helm-charts#1621]
**Describe the feature:**
I want the ability to configure a default indexing pattern (or patterns) in the kibana helm-chart
so that I can avoid having to manually configure index patterns every time I create a new kubernetes cluster with kibana.

**Describe a specific use case for the feature:**
I have code created that creates a k8s cluster with elastic/kibana/fluentd on it.  Every single time I create a cluster, when I navigate to kibana, it always asks me to configure an index pattern (because one does not exist).  I know what this pattern is and would like to define that in the helm chart so that it gets configured for me during the deployment process.

**Comment by v8v5v** [comment]
Hello, @DotNetRockStar 
I am not maintainer, just a user. I am also trying to automate ELK cluster bootstrap.
Today I learned that there is a kibana API for this purpose: https://www.elastic.co/guide/en/kibana/7.17/index-patterns-api-create.html
I hope, it might be helpful for your issue.

**Comment by framsouza** [comment]
It doesn't seem to an issue with helm-chart itself. please, for questions like that use our discuss [https://discuss.elastic.co](https://discuss.elastic.co)

**Comment by DotNetRockStar** [comment]
I disagree with the hcomment, but I understand. I will proceed with another solution.
